lib/engine/game/g_1880/step/dividend.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/dividend'
+require_relative '../../../step/minor_half_pay'
+
+module Engine
+  module Game
+    module G1880
+      module Step
+        class Dividend < Engine::Step::Dividend
+          def actions(entity)
+            return [] if entity.minor?
```

Comment:
You have both ways of doing minor?

Reply:
not sure what you're asking. what do you mean both ways?

Reply:
The next line where using corporation, with type minor. So are both ways of doing minors used?

Reply:
fixed